js/ui/layout.js

```diff
@@ -309,6 +309,11 @@ const LayoutManager = new Lang.Class({
         this.emit('hot-corners-changed');
     },
 
+    setViewsClone: function(actor) {
+        this._viewsClone = actor;
```

Comment:
Perhaps should also initialized to `null` in the constructor, like in the layout manager class? (Or remove it from there too.)